annotations.md

```diff
@@ -0,0 +1,25 @@
+# Annotations
+This OPTIONAL property MUST use the [annotation rules](annotations.md#rules).
```

Comment:
imho something like "Several components of the specification, like Image Manifests and Descriptors, feature an optional _annotations_ property, whose format is common and defined in this section"